packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Java.ps1

```diff
@@ -195,20 +181,28 @@ Function Get-Java
     # Shell arguments for the utility classes e.g. Import, Shell
     if ($PsCmdlet.ParameterSetName -eq 'UtilityInvoke')
     {
-      # Get the commandline args
+      # Generate the commandline args
       $RepoPath = Join-Path  -Path $Neo4jServer.Home -ChildPath 'lib'
       #  Get the default classpath jars
-      $ClassPath = ''    
+      $ClassPath = ''
+      # Enumerate all JARS in the Repo directory and add to the class path
       Get-ChildItem -Path $RepoPath | Where-Object { $_.Extension -eq '.jar'} | % {
         $ClassPath += "`"$($_.FullName)`";"
       }
+      # Enumerate all JARS in the bin directory and add to the class path
+      Get-ChildItem -Path (Join-Path  -Path $Neo4jServer.Home -ChildPath 'bin') | Where-Object { $_.Extension -eq '.jar'} | % {
+        $ClassPath += "`"$($_.FullName)`";"
+      }
       if ($ClassPath.Length -gt 0) { $ClassPath = $ClassPath.SubString(0, $ClassPath.Length-1) } # Strip the trailing semicolon if needed
 
       $ShellArgs = @()
-      if ($Env:JAVA_OPTS -ne $null) { $ShellArgs += $Env:JAVA_OPTS }
-      if ($Env:EXTRA_JVM_ARGUMENTS -ne $null) { $ShellArgs += $Env:EXTRA_JVM_ARGUMENTS }
-      $ShellArgs += @("-classpath $($Env:CLASSPATH_PREFIX);$ClassPath","-Dapp.repo=`"$($RepoPath)`"","-Dbasedir=`"$($Neo4jServer.Home)`"")
-      
+      if ($EnvJavaOpts -ne $null) { $ShellArgs += $EnvJavaOpts }
+      $ShellArgs += @("-classpath $($EnvClassPrefix);$ClassPath",
+                      "-Dapp.repo=`"$($RepoPath)`"", `
+                      "-Dbasedir=`"$($Neo4jServer.Home)`"", `
+                      '-Dfile.encoding=UTF-8')
+      # TODO Do I need to add -Dapp.pid="$$" ?
```

Comment:
No! Nor `app.repo` and `app.name`. You'll find I've removed all three already -- sorry about the merge.